src/main/java/org/jabref/logic/importer/fetcher/ComplexSearchQuery.java

```diff
@@ -135,6 +142,21 @@ public ComplexSearchQueryBuilder journal(String journal) {
             return this;
         }
 
+        public ComplexSearchQueryBuilder terms(Collection<Term> terms) {
```

Comment:
I would add this directly as a constructor, or if you want to have a more descriptive name as a `static ComplexSearchQuery fromTerms(Collection<Term> terms)` method in `ComplexSearchQuery`. The builder pattern is only useful if you have a lot of possible constructor arguments, and want to create a nice interface for it. I actually think the builder interface is mostly outdated since IDEs are now intelligent enough to show named constructor arguments, e.g
![image](https://user-images.githubusercontent.com/5037600/91430460-3cee2a80-e85f-11ea-8186-bdb78263e5c8.png)